### PR TITLE
Feat/unique team reviewers

### PR DESCRIPTION
**Pull request by dlaird-ovo**
## Hey, I just made a Pull Request!

When creating a PR from a software template you may wish to add from Groups within the catalog AND add a default team.  But you may find that the Group you select is the default team and then you have duplicates.  

```yaml
teamReviewers:
  - Team A
  - Team B
  - Team A
```

The API then fails to add any teamReviewers to the PullRequest.  This change makes the array unique before calling API so that this use-case is handled.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Review by awanlin — APPROVED** [review]
Thanks for this @dlaird-ovo, looks good!

File: .changeset/fair-spies-rescue.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend-module-github': minor
```
Comment (benjdlambert) [inline .changeset/fair-spies-rescue.md:2]:
```suggestion
'@backstage/plugin-scaffolder-backend-module-github': patch
```

**Review by benjdlambert — CHANGES_REQUESTED** [review]
Just a small change on the changeset, but this looks good! 

`patch` is used as `minor` when the version of the package is 0.x

**Review by benjdlambert — APPROVED** [review]
Nice! Thanks :tada:

**Pull request merged**